### PR TITLE
Adds an actual sprite for the singularity hammer.

### PR DESCRIPTION
**Pull request by LordShekelstein**
## What Does This PR Do
Adds a unique sprite for the singularity hammer as it currently uses the same one as mjolnir. 

## Why It's Good For The Game
Adds an identifier for the singularity hammer as opposed to it being the same sprite used by the mjolnir. May give players some incentive to use the singularity hammer. 

## Images of changes
SingularityHammer (1)
unknown (8)
unknown (7)
unknown (6)
unknown (4)
unknown (3)
unknown (2)
unknown (1)
unknown

## Changelog
:cl:
add: A sprite for the singularity hammer
fix: Mjolnir and Singularity Hammer using the exact same sprites.
/:cl:

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
That is one beautiful sprite.

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -166,7 +166,7 @@
 	var/mob/living/M = loc
 
 	if(M.a_intent in accepted_intents)
-		if(istype(M.mind.martial_art, /datum/martial_art/cqc))
```
Comment (AffectedArc07) [inline code/modules/clothing/gloves/miscellaneous.dm:169]: Youve butchered your branch here

Comment (LordShekelstein) [reply code/modules/clothing/gloves/miscellaneous.dm:169]: Yeah I'm not too sure what happened here. According to the more mentally well-equipped coders my VSC was being funky. I'm not trying to sneak in any naughty code.

**Comment by SteelSlayer** [comment]
Looks good. There are some small issues with some the inhand icons though, such as this:

image

The staff sprite ends before it gets to the hand.

**Comment by LordShekelstein** [comment]
image

Right, all fixed up now.

**Review by Dumbdumn5 — APPROVED** [review]
Holy SHIT that's so fucking cool. 👍

**Comment by S34NW** [comment]
In your example images, the bottom and 3rd from bottom images show a slightly different shade of grey/black on the sprite. Is this just screenshot compression?

**Comment by LordShekelstein** [comment]
Originally it was the wrong shade of black. However, I've changed this since the original screenshot and was simply too lazy to update the photo. 
image

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**